install_config/web_console_customization.adoc

```diff
@@ -174,6 +174,36 @@ assetConfig:
 ====
 endif::[]
 
+[[enable-tech-preview-feature]]
+=== Enabling Features in Tech Preview
+
+Sometimes features will be available in tech preview, by default these features may be hidden
+from end users.  
+
+The current list of features in tech preview is:
+
+* pipelines - Enabling this will add the Pipelines navigation item underneath the Builds menu
```

Comment:
@bfallonf It's not just listing the features. It's telling you the property names you need to use in this config to turn them on. (We can adjust the wording here.)

Agree it should be added to the release notes.